README.md

```diff
@@ -5,27 +5,51 @@
 [![Test Coverage](https://codeclimate.com/github/18F/federalist/badges/coverage.svg)](https://codeclimate.com/github/18F/federalist/coverage)
 [![Dependency Status](https://gemnasium.com/badges/github.com/18F/federalist.svg)](https://gemnasium.com/github.com/18F/federalist)
 
-***Under active development. Everything is subject to change. Learn more at the [documentation site](https://federalist-docs.18f.gov/). Interested in talking to us? [Join our public chat room](https://chat.18f.gov/).***
+***Federalist is updated regularly.  Contact us via [email](federalist-support@gsa.gov) or [Join our public chat room](https://chat.18f.gov/) to talk to us and stay informed. Or, check our our [documentation](https://federalist-docs.18f.gov/).***
 
-Federalist is a unified interface for publishing static government websites. It automates common tasks for integrating GitHub and Amazon Web Services to provide a simple way for developers to launch new static websites or more easily manage existing static websites. This repo is home to "federalist-core" - a Node.js app that allows government users to add and configure their Federalist sites.
+## About Federalist
+
+Federalist helps government agencies publish basic websites quickly and seamlessly.  It provides a simple interface for managing simple website content that is compliant with federal guidelines.  More complex tasks are automated behind the scenes, providing a way for users to more easily create and launch government websites.  Federalist is built on top of Github and Amazon Web Services.   
+
+## Examples
+
+Partner agencies across the federal government use Federalist to host websites.  A few examples include: 
+
+- [College Scorecard] (https://collegescorecard.ed.gov/)
+- [Natural Resources Revenue Data] (https://revenuedata.doi.gov/)
+- [NSF Small Business Innovation Research program] (https://seedfund.nsg.gov)
+
+More examples can be found [here] (https://federalist-docs.18f.gov/pages/about-federalist/example-sites/). 
 
 ## Getting started
 
+This section will walk you through how to install Federalist locally onto your own machine or within your own environment.  Depending on your familiarity with the dependencies and process, this may take anywhere between an hour or two to the better part of a day.  If you need help, [contact us](federalist-support@gsa.gov). 
+
 ### Dependencies / Tooling
 
 Before you start, ensure you have the following installed:
 
 - [Cloud Foundry CLI](https://docs.cloudfoundry.org/cf-cli/install-go-cli.html)
 - [Docker Compose](https://docs.docker.com/compose/install/#install-compose)
 
+Most of these dependencies can be installed relatively quickly if you don't already have them.  Most can also be installed via terminal using popular package managers like [brew](https://brew.sh/). 
```

Comment:
I think this is out of date since now there are only two requirements. The word `most` feels a little too "big." I think we could just remove these sentences.

Reply:
Super.  We can edit.

Reply:
removing